tests/runtest

```diff
@@ -0,0 +1,36 @@
+SHELL := /bin/bash
```

Comment:
What do you think about naming this file `Makefile` and putting it in the root of the repo?

It is sort of a standard and many developers will recognize it.

The file itself is 💣

Reply:
Yeah, this is a good change. Done: [7d5c780](https://github.com/ivica-k/terraform-validate-explorer/pull/13/commits/7d5c780e3122ef97dc2b124b4a55e120de5ecc28)